src/test/java/seedu/ezdo/model/task/TaskTest.java

```diff
@@ -21,4 +21,12 @@ public void hashCode_isEqual() throws Exception {
         assertTrue(task.hashCode() == task.hashCode());
     }
 
+    @Test
```

Comment:
@e0003133 @JTLX eh I not sure if tests can be done like this? I thought always need to input a command followed by assertion?